src/dbmanager.cpp

```diff
@@ -277,20 +278,22 @@ bool DBManager::updateNote(NoteData* note)
     int id = note->id();
     qint64 epochTimeDateModified = note->lastModificationdateTime().toMSecsSinceEpoch();
     QString content = note->content()
-            .replace("'","''")
+            .replace(QLatin1String("'"), QLatin1String("''"))
             .replace(QChar('\x0'), emptyStr);
     QString fullTitle = note->fullTitle()
-            .replace("'","''")
+            .replace(QLatin1String("'"), QLatin1String("''"))
             .replace(QChar('\x0'), emptyStr);
```

Comment:
Not sure why we need this? Was it also related to it breaking the SQL query, so possibly resolved with `bindValue`?

Reply:
Not sure about it myself.
Removing it doesn't cause any difference in the output. But maybe there's some specific case against which this needs to be tested or was put in, in the first place by @theshadowx